data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java

```diff
@@ -143,11 +159,11 @@ private void registerControllers(ServiceExtensionContext serviceExtensionContext
                 resourceDescriptionRequestHandler,
                 connectorDescriptionRequestHandler);
 
-        List<Handler> handlers = new LinkedList<>();
+        LinkedList<Handler> handlers = new LinkedList<>();
```

Comment:
can't be a standard `ArrayList` implementation? like `var handlers = new ArrayList<Handler>()`?

Reply:
Don't know. 
As PR is quite big already I prefer to let it as it is for now to avoid any side-effect.

Reply:
Ok, but at least a `var` can be used instead of define the type explicitly